### PR TITLE
Supporting PyPi Upload Via Travis

### PR DESCRIPTION
**Pull request by idjaw**
-Updating travis config to allow upload to pypi on tags
-Removed some templated items from README

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,5 +13,13 @@ before_install:
 install:
   - pip install tox-travis
 
-script: tox
+script: tox -r
```
Comment (fbouliane) [inline .travis.yml]: why ? (non-blocking)

File: README.rst [hunk 2]
```diff
@@ -2,18 +2,12 @@
...
+(PDUs).
 
-Please fill here a long description which must be at least 3 lines wrapped on
-80 cols, so that distribution package maintainers can use it in their packages.
-Note that this is a hard requirement.
+virtualpdu is licensed under the Apache License like the rest of OpenStack.
 
-* Free software: Apache license
+* Free software: Apache license, Version 2.0
 * Documentation: http://docs.openstack.org/developer/virtualpdu
```
Comment (fbouliane) [inline README.rst:11]: These urls are not existing yet ?

**Comment by fbouliane** [comment]
:+1:

File: README.rst (hunk 2 above)
Comment (idjaw) [reply README.rst:11]: Not yet.

File: .travis.yml (hunk 1 above)
Comment (idjaw) [reply .travis.yml]: I guess -r is run when it realizes there isn't a tox environment. So it probably wasn't necessary. Did it out of habit.

**Pull request merged**